### PR TITLE
minor fixes for svcat sync broker command

### PR DESCRIPTION
**Pull request by jboyd01**
fix issues from https://github.com/kubernetes-incubator/service-catalog/pull/2315#pullrequestreview-161797490

- updated help text
- observe error from ApplyScopedFlags
- comment fix
- print error when broker can't be found

### CODE REVIEW

File: pkg/svcat/service-catalog/broker.go [hunk 1]
```diff
@@ -239,14 +240,14 @@ func (sdk *SDK) Sync(name string, scopeOpts ScopeOptions, retries int) error {
...
-			broker, err := sdk.RetrieveBroker(name)
+			var broker *v1beta1.ClusterServiceBroker
+			broker, err = sdk.RetrieveBroker(name)
 			if err == nil {
 				broker.Spec.RelistRequests = broker.Spec.RelistRequests + 1
 
 				_, err = sdk.ServiceCatalog().ClusterServiceBrokers().Update(broker)
 				if err == nil {
 					success = true
-					fmt.Printf("synced clusterservicebroker (%s)\n", name)
```
Comment (shubheksha) [inline pkg/svcat/service-catalog/broker.go]: Why are we removing this print statement here?

Comment (jboyd01) [reply pkg/svcat/service-catalog/broker.go]: @shubheksha I should have left a comment or note about this in the PR. 
I went to correct the text (caught it on a late code review) and then found we were already printing a good message in the calling function:

https://github.com/kubernetes-incubator/service-catalog/blob/f3a7a072ac60d9ccb7bd55b05ada0de18db3cab1/cmd/svcat/broker/sync_cmd.go#L75

I verified before and after removing this line, it was printing both messages.

File: cmd/svcat/command/command.go [hunk 2]
```diff
@@ -41,7 +41,10 @@ func PreRunE(cmd Command) func(*cobra.Command, []string) error {
 			nsCmd.ApplyNamespaceFlags(c.Flags())
 		}
 		if scopedCmd, ok := cmd.(HasScopedFlags); ok {
-			scopedCmd.ApplyScopedFlags(c.Flags())
+			err := scopedCmd.ApplyScopedFlags(c.Flags())
```
Comment (carolynvs) [inline cmd/svcat/command/command.go]: 🎉 Thanks for catching this!

**Pull request merged**